src/promptflow-core/promptflow/core/_serving/app_base.py

```diff
@@ -55,7 +56,15 @@ def init_app(self, **kwargs):
         self.sample = get_sample_json(self.project_path, logger)
 
         self.init = kwargs.get("init", {})
-        logger.info("Init params: " + str(self.init))
+        try:
+            init_params = os.environ.get(PROMPTFLOW_FLOW_INIT_CONFIG, "{}")
```

Comment:
only when the init param is empty,  we need to try loading from the environment variables.

Reply:
ok, done.